test/config.js

```diff
@@ -32,6 +32,7 @@ export default {
     }
   },
   daemon1: {
+    silent: true,
```

Comment:
This turns off IPFS console output upon starting the IPFS instance so the swarm addresses and other output is not shown anymore in the tests making it cleaner to read test run logs.